sqlobject/src/test/java/org/jdbi/v3/sqlobject/locator/TestSqlLocator.java

```diff
@@ -32,7 +33,7 @@
     public PgDatabaseRule dbRule = new PgDatabaseRule().withPlugin(new SqlObjectPlugin());
 
     @Test
-    public void testLocateConfigDriven() {
+    public void testLocateConfigDriven() throws Exception {
```

Comment:
Why does this throw an exception now?

Reply:
I think Travis isn't actually testing against 11 because of toolchains...